[xcode11-feature] [MediaPlayer] AVMediaSelectionGroup & AVMediaSelectionOption not available on Watch

<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨 -->

<!--
Please fill in the *entire* template below.
The template is designed to avoid unnecessary delays to confirm and fix reported issues. Issues with missing information generally end up with a `need-info` label and take significantly more time to fix. Please help us help you!
For more information see https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions
For support requests use https://www.xamarin.com/support
-->
### Steps to Reproduce

1. Use AVMediaSelectionGroup or AVMediaSelectionOption on watch
-Dependent on AVFoundation

### Expected Behavior
To be able to use it

### Actual Behavior
Says that the type or namespace does not exist in the namespace AVFoundation 

### Environment
xcode11

<!--
1.
Visual Studio: Help > About Microsoft Visual Studio > Copy Info [button]
Visual Studio for Mac: Visual Studio > About Visual Studio > Show Details > Copy Information [button]
2. Paste into the code block below (between ```)
-->

```
```

### Build Logs

<!--
1. Place cursor below this comment block.
2. Attach build log or link to gist (https://gist.github.com/) of the log.
-->
build/watch/watch/MediaPlayer/AVMediaSelectionGroup_MPNowPlayingInfoLanguageOptionAdditions.g.cs(41,120): error CS0234: The type or namespace name 'AVMediaSelectionGroup' does not exist in the namespace 'AVFoundation' (are you missing an assembly reference?)
build/watch/watch/MediaPlayer/AVMediaSelectionOption_MPNowPlayingInfoLanguageOptionAdditions.g.cs(41,110): error CS0234: The type or namespace name 'AVMediaSelectionOption' does not exist in the namespace 'AVFoundation' (are you missing an assembly reference?)
make[1]: *** [build/watch/watch-32/Xamarin.WatchOS.dll] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [all-recurse] Error 1

### Example Project (If Possible)

<!--
1. Place cursor below this comment block.
2. Drag and drop the compressed project or files needed to reproduce.
-->

<!--
Switch to the "Preview" tab to ensure your issue renders correctly.
-->

--- a/src/mediaplayer.cs
+++ b/src/mediaplayer.cs
@@ -20,10 +20,6 @@ using AppKit;
 using UIKit;
 #endif
 using System;
-#if WATCH
-using AVMediaSelectionGroup = Foundation.NSObject;
-using AVMediaSelectionOption = Foundation.NSObject;
-#endif
 
 namespace MediaPlayer {
 #if XAMCORE_2_0 || !MONOMAC

--- a/tests/xtro-sharpie/watchOS-MediaPlayer.todo
+++ b/tests/xtro-sharpie/watchOS-MediaPlayer.todo
@@ -1,2 +1,0 @@
-!missing-selector! AVMediaSelectionGroup::makeNowPlayingInfoLanguageOptionGroup not bound
-!missing-selector! AVMediaSelectionOption::makeNowPlayingInfoLanguageOption not bound

